test/utils/writeTypingsHelper.js

```diff
@@ -26,21 +26,19 @@ module.exports = {
     });
     return dir;
   },
-  dynamic: (rootTestPath, context, { elementsMap, widgets, dependencies }) => {
+  dynamic: (rootTestPath, context, dynamicTypingsFiles) => {
```

Comment:
i would still make "dynamicTypingsFiles" an array of ({content, fileName}), where content is always a string (not an array).
no need to know all of that here

Reply:
Yeah but that is how the data is store and processed.
We can a bit of that in the positive/negative.spec files that uses this method but then we'd be writing the same code twice which is what this function was meant to solve.

Reply:
So I'd rather just leave it like that.
This way we can also introduce new typings concepts with ease, just add the key

Reply:
i'm not sure i follow.... why doesn't `getDynamicTypings` just flatten its return value ?